packages/o2-blocks/babel.config.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+	extends: require.resolve( '@automattic/calypso-build/babel.config.js' ),
+	presets: [ require( '@automattic/calypso-build/babel/wordpress-element' ) ],
+	env: {
+		build_pot: {
+			plugins: [ '@wordpress/babel-plugin-makepot' ],
```

Comment:
Default options should be fine here, see https://github.com/WordPress/gutenberg/blob/45da3cf1948aa2723bfb649dae37287ca2495d64/packages/babel-plugin-makepot/src/index.js#L40-L48